Automated pipeline for air quality

hristogichev-akros opened the pull request.

Commit:
automate air quality data ingested by pipeline

Commit:
add raw data

Commit:
gitignore

Commit:
Merge branch 'main' into pipeline-loading-data-postgres

Commit:
fix issues

Commit:
fix lint issues

Commit:
fix lint issues insert_stations

Commit:
fix lint issues whithespace

Commit:
remove lint issues

Commit:
remove files that are not required for tracking

Akros-STM reviewed (commented):
Could you have a look ath the comments?

Commit:
Remove ignored files from git tracking

- Remove .file_versions/ directories (Mage AI version control files)
- Remove .variables/ directories (Mage AI variable cache files)
- Remove .operation_history/ directories (Mage AI operation history files)
- Remove .logs/ directories (Mage AI log files)
- Remove .DS_Store files (macOS system files)

These files should not be tracked in git according to .gitignore rules.

Akros-STM reviewed (commented):
would you have a look again?

hristogichev-akros commented on deployment/postgres/script/ddl-nebel.sql:
```diff
@@ -189,18 +189,18 @@ CREATE TABLE IF NOT EXISTS station_measurements (
 );
 
 -- Abgeleitete tägliche Aggregation (Median/Mean etc.) – optional, aber häufig praktisch
```
yes

hristogichev-akros commented on deployment/mage/data/stations.csv:
```diff
@@ -0,0 +1,17 @@
+Station,Tag,Easting,Northing,Meters_Above_Sealevel,Locationtype,Remarks
```
yes, the new functionality use the raw_data folder for legacy reazon i not deleted the other folder cause it can bu used for old python files is up to you to decide when to remove legacy files

Commit:
rename nabel file

Akros-STM reviewed (dismissed):
Thanks for this project increment!

Commit:
Remove .ssh_tunnel directory from git tracking

This directory should be ignored according to .gitignore rules.

Akros-STM reviewed (approved):
Thanks!
